tools/sctool/src/main/java/com/horizen/ScBootstrappingToolCommandProcessor.java

```diff
@@ -393,27 +402,40 @@ private void processGenerateCertWithKeyRotationProofInfo(JsonNode json) throws E
                 return;
             }
 
-            mastersPublicKeys.add(pkNode.asText());
+            String masterKey = pkNode.asText();
+
+            int duplicateIndex = mastersPublicKeys.indexOf(masterKey);
+            if (duplicateIndex != -1) {
+                printGenerateCertWithKeyRotationProofInfoUsageMsg(String.format("duplicated keys in master keys are found. MasterKey with index %d equals to masterKey with index %d", duplicateIndex + 1, index));
```

Comment:
Consider changing error to:

"masterKeys contains duplicate values. MasterKey with index %d equals to masterKey with index %d"